src/pytorch_lightning/loggers/csv_logs.py

```diff
@@ -49,8 +49,8 @@ class ExperimentWriter:
     NAME_METRICS_FILE = "metrics.csv"
 
     def __init__(self, log_dir: str) -> None:
-        self.hparams = {}
-        self.metrics = []
+        self.hparams: dict = {}
+        self.metrics: list = []
```

Comment:
```suggestion
        self.hparams: Dict[str, any] = {}
        self.metrics: List[Dict[str, float]] = []
```
If I'm not mistaken. Or perhaps if float is too specific, it would have to become Any.

Reply:
updated in https://github.com/Lightning-AI/lightning/pull/13538/commits/1582c63d7704079965c1f95466a9ecfdb9d1a389